### ISSUE TITLE
Doc for version 1.0.0-rc.1

### ISSUE THREAD

**Issue by jampack** [tymondesigns/jwt-auth#1340]
is there any documentation available for version 1.0.0-rc.1 if we want to implement the latest version?

**Comment by kinyou** [comment]
1.   composer require tymon/jwt-auth 1.0.0-rc.1
2.   add  something just like this  config/app.php
      Tymon\JWTAuth\Providers\LaravelServiceProvider::class,


      #aliases - array
     'JWTAuth' => Tymon\JWTAuth\Facades\JWTAuth::class,
     'JWTFactory' => Tymon\JWTAuth\Facades\JWTFactory::class,


3. php artisan vendor:publish --provider="Tymon\JWTAuth\Providers\LaravelServiceProvider"

4. php artisan jwt:secret

5. last good luck with you , enjoy it!

**Comment by jampack** [comment]
@kinyou and rest of the document applies the same for newer version? i dont think so.

**Comment by kinyou** [comment]
@akkhan20 yes you are right , the <tymon/jwt-auth 1.0.0-rc.1> version is only use in laravel 5.5+

**Comment by philliperosario** [comment]
The 1.0.0-rc.1 version requires you to implement Tymon\JWTAuth\Contracts\JWTSubject on your user model too. You must then add the required methods, which are getJWTIdentifier() and getJWTCustomClaims().

```
public function getJWTIdentifier()
{
    return $this->getKey(); \\ Eloquent Model method
}

public function getJWTCustomClaims()
{
    return [];
}
```

See: #260

**Comment by philliperosario** [comment]
#1355
